TCAT/Cells/PlaceTableViewCell.swift

```diff
@@ -8,12 +8,15 @@
 
 import UIKit
 
-class SearchResultsCell: UITableViewCell {
-    let labelWidth: CGFloat = 45.0
+class PlaceTableViewCell: UITableViewCell {
+    
+    var place: Place?
```

Comment:
I'm hesitant about giving the cell view a reference to the model it shows, is storing it here necessary?

Reply:
@drewsdunne At the moment I only use it so auto configure what color for the pin based on whether it's a bus stop or not. The cell is the same except slight aesthetic differences based on the type of the place, what do you recommend?

Reply:
I usually just provide a function such as setup(for place: Place) and call that tableView function but neither is really a great MVC solution :( so whatever works.